src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingLumberjack.java

```diff
@@ -95,6 +105,32 @@ public BuildingLumberjack(final IColony c, final BlockPos l)
         super(c, l);
 
         keepX.put(itemStack -> ItemStackUtils.hasToolLevel(itemStack, ToolType.AXE, TOOL_LEVEL_WOOD_OR_GOLD, getMaxToolLevel()), new Tuple<>(1, true));
+
+        if (recipes.isEmpty()) {
+            addStrippedWoodRecipe(Items.OAK_LOG, Items.STRIPPED_OAK_LOG);
```

Comment:
I haven't looked at the Log code and etc, but i would assume there's some way, either in the block or otherwise, to get the stripped version of a log from a log. Doing this would add better support for modded logs

Reply:
Agreed on that. We should probably do something like Items.Tags.Logs.getItems().forEach(addRecipe)

Reply:
Tbh, I can't find anything. The only mention of that in forge is in the item.AxeItem, and there it's also done manually:
`   protected static final Map<Block, Block> BLOCK_STRIPPING_MAP = (new Builder<Block, Block>()).put(Blocks.OAK_WOOD, Blocks.STRIPPED_OAK_WOOD).put(Blocks.OAK_LOG, Blocks.STRIPPED_OAK_LOG).put(Blocks.DARK_OAK_WOOD, Blocks.STRIPPED_DARK_OAK_WOOD).put(Blocks.DARK_OAK_LOG, Blocks.STRIPPED_DARK_OAK_LOG).put(Blocks.ACACIA_WOOD, Blocks.STRIPPED_ACACIA_WOOD).put(Blocks.ACACIA_LOG, Blocks.STRIPPED_ACACIA_LOG).put(Blocks.BIRCH_WOOD, Blocks.STRIPPED_BIRCH_WOOD).put(Blocks.BIRCH_LOG, Blocks.STRIPPED_BIRCH_LOG).put(Blocks.JUNGLE_WOOD, Blocks.STRIPPED_JUNGLE_WOOD).put(Blocks.JUNGLE_LOG, Blocks.STRIPPED_JUNGLE_LOG).put(Blocks.SPRUCE_WOOD, Blocks.STRIPPED_SPRUCE_WOOD).put(Blocks.SPRUCE_LOG, Blocks.STRIPPED_SPRUCE_LOG).build();
`

Reply:
How does vanilla determine that one block can be made to the stripped version by using an axe on it? Hardcoded?

Reply:
If I understand correctly, it simply uses the map I posted above:
`
Block block = BLOCK_STRIPPING_MAP.get(blockstate.getBlock());
      if (block != null) {
         PlayerEntity playerentity = context.getPlayer();
         world.playSound(playerentity, blockpos, SoundEvents.ITEM_AXE_STRIP, SoundCategory.BLOCKS, 1.0F, 1.0F);
`
Unfortunately, that map is protected and there's no Getter that I can see.

Reply:
I checked with forge and that's the only way to do it.